pkg/analyser/analyzer.go

```diff
@@ -82,28 +82,28 @@ func (a Analyzer) Analyze(remoteResources, resourcesFromState []resource.Resourc
 		analysis.AddManaged(stateRes)
 
 		delta, _ := diff.Diff(stateRes, remoteRes)
-		if len(delta) > 0 {
-			sort.Slice(delta, func(i, j int) bool {
```

Comment:
I think this is needed but it's missing sort on the path, only sorting on type is not sufficient

Reply:
My tests showed that len(delta) is always equal to 1 so the sort has no effect at all. Would be nice if someone could confirm that

Reply:
no you could have multiple diff on one resource so the len will be greater